docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -13,6 +13,48 @@ ms.devlang: azurecli
 
 # Azure CLI release notes
 
+## July 30, 2019
+
+Version 2.0.70
+
+### ACR
+
+* Fixed issue #9952 (a regression in the `acr pack build` command).
+* Removed the default builder image name in `acr pack build`.
+
+### Appservice
```

Comment:
This line should as the following format:
### Appservice

It looks missing an empty line behind ### Appservice.